Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -267,7 +267,7 @@ void MemWatchWidget::onMemWatchContextMenuRequested(const QPoint& pos)
     contextMenu->addSeparator();
   }
 
-  if (!node || node->isGroup())
+  if (node == m_watchModel->getRootNode() || node->isGroup())
```

Comment:
_Ideally_, this change (which relates to https://github.com/aldelaro5/dolphin-memory-engine/pull/175) should land in its own PR.